### PR TITLE
Feature/fix img

### PR DESCRIPTION
**Pull request by andriikamaldinov1**

### CODE REVIEW

File: open-source/sub-header/sub-header.component.html [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
-                <span>All Stars</span>
-                <img
-                    width="8"
-                    height="8"
-                    [ngSrc]="'rectangle' | asset"
-                    alt="rectangle"
-                    class="rectangle" />
-            </button>
-            <button class="stars-btn row full-white span bg-black">1104</button>
+            <a href="https://github.com/JsDaddy/">
```
Comment (xakeppok) [inline open-source/sub-header/sub-header.component.html:5]: use target blank for external links

File: open-source/sub-header/sub-header.component.html [hunk 2]
```diff
@@ -2,17 +2,21 @@
...
+                    <img
+                        width="8"
+                        height="8"
+                        [ngSrc]="'rectangle' | asset"
+                        alt="rectangle"
+                        class="rectangle" />
+                </button>
+            </a>
+            <a href="https://github.com/JsDaddy/">
+                <button class="stars-btn row full-white span pointer bg-black">1104</button>
```
Comment (xakeppok) [inline open-source/sub-header/sub-header.component.html:18]: data should be from github APUI. look at company site. it looks like it reusable component

**Review by GlebChiz — APPROVED** [review]

File: open-source/accordion/accordion.component.scss [hunk 3]
```diff
@@ -119,13 +83,17 @@
...
                         border-width: 2px 0 0 0;
                     }
-                    &:after {
-                        transition: all 1s;
-                        content: url('/assets/images/yellow-chevron-up.svg');
-                    }
                     .accordion-btn {
                         color: var(--yellow);
                     }
+                    .right-side-accordion {
```
Comment (xakeppok) [inline open-source/accordion/accordion.component.scss:89]: .right-side-accordion {
    .chevron {
        &-default {
            display: none;
        }
        &-active {
            display: block;
        }
    }
}
maybe in this way? what do you think?

File: open-source/header/header.component.html [hunk 4]
```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="center-side row ai-c" *ngFor="let items of headerDoc; trackBy: trackByPath">
```
Comment (xakeppok) [inline open-source/header/header.component.html:9]: maybe item?

File: open-source/sub-header/sub-header.component.scss [hunk 5]
```diff
@@ -5,23 +5,48 @@
...
-                .rectangle {
-                    position: relative;
-                    left: 15px;
-                    top: 6px;
+            .github-info {
+                display: flex;
+                margin: 0 44px 0 20px;
+
+                > div {
+                    display: flex;
```
Comment (xakeppok) [inline open-source/sub-header/sub-header.component.scss:13]: we have it in general styles. please reuse it

**Pull request merged**